java/amazon-kinesis-producer/src/main/java/com/amazonaws/services/kinesis/producer/LogInputStreamReader.java

```diff
@@ -46,13 +44,13 @@
         emitters.put("trace", new LoggingFunction() {
             @Override
             public void apply(String message) {
-                log.trace(message);
+                log.info(message);
             }
         });
         emitters.put("debug", new LoggingFunction() {
             @Override
             public void apply(String message) {
-                log.debug(message);
+                log.info(message);
```

Comment:
Why are we changing this to `info` from `debug`? Looks like these are supposed to match the corresponding key.

Reply:
Java layer currently doesn't have logging level configuration support and uses default logging level of info. We only modify the logging level for the native layer as of now since that is where all of the external interactions and SDK live. Hence changed this to info so that we do log the trace and debug logging when enabled from native layer.